### PR TITLE
Documentation about good practices

### LINKED ISSUE #146: Guidelines about good practices

### PR DESCRIPTION
**Pull request by driccio**
Resolves #146

@Lysoun @nroulon @asolovieff

### CODE REVIEW

File: docs/good-practices/delete/delete.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  - update the code to remove the property/column.
+  - create an AFTER script that remove the property/column.
+
+## Used
+
+- In V+1 :
+  - update the code to mark this property/column as deprecated.
+  - update the code to stop the reads on this property/column.
+    - no code must use this property/column for treatment.
+    - keep writes because version V use this property/column for treatment.
```
Comment (xhanin) [inline docs/good-practices/delete/delete.md]: hope the treatment can heal the app :)

Comment (asolovieff) [reply docs/good-practices/delete/delete.md]: Remplacer treatment par computation

File: docs/good-practice/update/update.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  - create a BEFORE script to update all entries using 'ENDED' to replace it with 'DONE'.
+  - update the code to remove 'ENDED' value.
+  - if your db has schema, create an AFTER script to remove 'ENDED' as possible value.
+  
+
+## Update type
+
+To proceed to an update type we can perform :
+- add a new temporary property from the one we want to rename
+- remove the original property
```
Comment (Lysoun) [inline docs/good-practice/update/update.md:38]: Is there a need to remove the property? I did such an operation on a mongo database and renaming my temporary property was enough to replace the old value with my new one.

Comment (Lysoun) [reply docs/good-practice/update/update.md:38]: Actually I read it all and you did take explain everything so I'm resolving this conversation.

**Review by Lysoun — APPROVED** [review]
@driccio I've read everything and reworded quite a lot of the good practice explanations. I think a link could be missing from the README to finalise this pull request. I'm letting you read it all once again and merge.

**Review by driccio — COMMENTED** [review]
Thanks a lot @Lysoun for the review/rewording.

I just don't understand why you renamed the folder. "Good practice**s**" sounds good for me. By the way, thanks.

**Comment by Lysoun** [comment]
> Thanks a lot @Lysoun for the review/rewording.
> 
> I just don't understand why you renamed the folder. "Good practice**s**" sounds good for me. By the way, thanks.

Both seem alright. I picked 'good practice' because Linguee told me it was more frequent
https://www.linguee.fr/francais-anglais/traduction/bonnes+pratiques.html
https://www.deepl.com/translator#fr/en/bonnes%20pratiques

**Pull request merged**